feature/src/main/java/com/teamkkumul/feature/mygroup/mygroupdetail/MyGroupDetailFragment.kt

```diff
@@ -246,11 +248,14 @@ class MyGroupDetailFragment :
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
-        if (viewModel.isMeetUpIncludeMeSelected.value) {
-            switchToMeetUpIncludeMeState()
-        } else {
-            switchToAllMeetUpState()
+        viewLifecycleOwner.lifecycleScope.launch {
```

Comment:
리뷰가 좀 늦었는데
```suggestion
        viewLifecycleScope.launch {
```

만들어 놓은 확장함수 있습니다!

Reply:
아......;....맞다..